tests/Integration/DataManagementSystem/Test_Client_FTS3.py

```diff
@@ -41,6 +41,7 @@ def test_cancelNotFoundJob(fts3db, fts3Client):
     Note: this test is not in the common tests because SQLte does not play nice with
     multiple table update, which is what is done in cancelNonExistingJob
     """
+    fts3db.createTables()
```

Comment:
That's a dirty hack, it clearly shouldn't be in a test
I think the bug is that the `FTS3DB` class should always call `createTables` and it isn't

Reply:
Then we have multiple bugs and dirty hacks, because:
- unit test _TestFTS3DB.py_ calls `createTables`
- for RequestDB we are in exactly the same situation.:
  - there's no `createTables` called in `RequestDB`
  - unit test _TestRequestDB.py_ calls `createTables`
  - integration test _TestReqDB.py_ is IIUC what creates the tables for RequestDB, through the fixture in the above unit test.

Reply:
You're right ! I just checked, and in the cases of the `RMS` and `FTS`, create table is called by the service (and so the tests), while the other systems call `createTable` directly in the initialization of the DB. I honestly don't know what the best of the two is...
However, in the tests you mention, the `createTable` is called in the fixture that basically does the same thing as the initialization of the handler. Not in the test itself